litecli/packages/special/dbcommands.py

```diff
@@ -27,14 +27,14 @@ def list_tables(cur, arg=None, arg_type=PARSED_QUERY, verbose=False):
         args = ("{0}%".format(arg),)
         query = """
             SELECT name FROM sqlite_master
-            WHERE type IN ('table','view') AND name LIKE ? AND name NOT LIKE 'sqlite_%'
+            WHERE type IN ('table','view') AND name LIKE ? AND name NOT LIKE 'sqlite_%' AND name NOT LIKE 'sqlean_define'
```

Comment:
Nit: We can be a bit more generic, like `sqlean_%`. Not blocking the PR though

Reply:
Good point. Done!